.gitignore

```
@@ -172,3 +172,9 @@ cython_debug/
 
 # PyPI configuration file
 .pypirc
+
+# ignore the blender install file as it is too large for github
+blender-4.3.2-linux-x64.tar.xz
+
+# removing the data directory
+data/
```

```suggestion
data/*
!data/.gitkeep

config/*
!config/.gitkeep
```
Create a .gitkeep file in the data and config folder then delete everything else inside it so that unnencessary files are ignored but the folder itself still gets tracked

---

Added the gitkeep files to both directories and updated the .gitignore to reflect that change